testing/tests/DevExpress.ui.widgets.scheduler/integration.appointments.tests.js

```diff
@@ -4039,12 +4043,13 @@ QUnit.module('Appointments', () => {
                     startDateExpr: 'startDateCustom',
                     endDateExpr: 'endDateCustom',
                     currentView: 'week',
-                    timeZone: 'Asia/Yekaterinburg',
+                    timeZone: 1,
```

Comment:
According to https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#timeZone, it should be `string`

Reply:
For more comfortable testing, we support an integer value in the code. Look at the calculateTimezoneByValue method.
In the test I need such a timezone, after applying which all the data will be displayed for one day and there will be no DST.